cherrypy/lib/cptools.py

```diff
@@ -586,6 +586,7 @@ def accept(media=None, debug=False):
 class MonitoredHeaderMap(_httputil.HeaderMap):
 
     def __init__(self):
+        super(MonitoredHeaderMap, self).__init__()
```

Comment:
Superclass's `__init__(*args, **kwargs)` has different signature. Please write test for this.

Reply:
Have taken the liberty of writing test case at cherrypy/test/test_misc_tools.py, please review

Reply:
still, you don't use args/kwargs, add them